src/vm/ceemain.cpp

```diff
@@ -982,20 +982,15 @@ void EEStartupHelper(COINITIEE fFlags)
             ClrSleepEx(g_pConfig->StartupDelayMS(), FALSE);
         }
 #endif
-
+        
 #if USE_DISASSEMBLER
         if ((g_pConfig->GetGCStressLevel() & (EEConfig::GCSTRESS_INSTR_JIT | EEConfig::GCSTRESS_INSTR_NGEN)) != 0)
         {
             Disassembler::StaticInitialize();
             if (!Disassembler::IsAvailable())
             {
-#ifdef HAVE_GCCOVER
-#ifdef _DEBUG
                 printf("External disassembler is not available. Disabling GCStress for GCSTRESS_INSTR_JIT and GCSTRESS_INSTR_NGEN.\n");
-#endif // _DEBUG
-                g_pConfig->SetGCStressLevel(
-                    g_pConfig->GetGCStressLevel() & ~(EEConfig::GCSTRESS_INSTR_JIT | EEConfig::GCSTRESS_INSTR_NGEN));
-#endif // HAVE_GCCOVER
+                IfFailGo(E_FAIL);
```

Comment:
If that is the case (fail) instead of disabling GCStress, the above message should be updated. And also I'd use error message handling or macro like ```fprinf(stderr,...)``` instead of normal printf.